subcommands/keys/rotate_targets.go

```diff
@@ -60,6 +61,13 @@ func doRotateTargets(cmd *cobra.Command, args []string) {
 	targetid, newCreds := replaceOfflineTargetKey(root, onlineTargetId, creds, keyType)
 	fmt.Println("= New target:", targetid)
 	removeUnusedKeys(root)
+	user, err := api.UserAccessDetails(factory, "self")
+	subcommands.DieNotNil(err)
+	root.Signed.Reason = &client.RootChangeReason{
+		PolisId:   user.PolisId,
+		Message:   "Target role key rotation",
```

Comment:
nitpick: a TUF role name is `targets`, not `target`.

Reply:
Sure, always make this mistake.